### PR TITLE
S2U-35 Lessons: View items based on grade

### PR DESCRIPTION
**Pull request by stetsche**
Jira: https://sakaiproject.atlassian.net/browse/S2U-35

Conversion script: https://github.com/sakaiproject/sakai-reference/pull/220

### CODE REVIEW

File: vuecomponents/bundle/src/main/bundle/condition_eu.properties [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+# Operators
+smaller_than = smaller than
+smaller_than_or_equal_to = smaller than or equal to
+equal_to = equal to
+greater_than_or_equal_to = greater than or equal to
+greater_than = greater than
+
+conjunction_and = Require this condition AND another condition
+conjunction_or = Require this condition OR another condition
```
Comment (stetsche) [inline vuecomponents/bundle/src/main/bundle/condition_eu.properties:33]: Hi @PabloSanRoman!
Could you please help me with this translations, when you have time? No rush.
I leave some screenshots here for context:

image
image
image

File: conditions/impl/pom.xml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        </dependency>
+        <!-- Hibernate dependencies -->
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-core</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <resources>
```
[the remark this answers is not in the document]
Comment (stetsche) [reply conditions/impl/pom.xml:75]: Removed the inclusion of test resources, .sql and .config files. I'll keep the .properties for the i18n bundle

**Comment by stetsche** [comment]
Thanks @frasese! I removed the mentioned sections in the pom and did a rebase.

**Comment by mpellicer** [comment]
Thanks to everybody that participated on this, it's an impressive piece of work. @frasese @jesusmmp @stetsche and Manu Fuster.

Just fix the codacy issues that are not risky to fix and we'll merge it.

**Comment by stetsche** [comment]
All codacy issues I didn't consider false alarms are now addressed and I squashed the commits. The PR should be ready to merge now.

**Comment by mpellicer** [comment]
MASTERPIECE

**Pull request merged**